client/src/ui/BitbakeCommands.ts

```diff
@@ -188,6 +190,25 @@ async function runTaskCommand (bitbakeWorkspace: BitbakeWorkspace, bitBakeProjec
   }
 }
 
+async function startToasterInBrowser (bitbakeDriver: BitbakeDriver): Promise<void> {
+  const DEFAULT_TOASTER_PORT = 8000
+  const command = 'nohup bash -c "source toaster start"'
+  const process = await runBitbakeTerminalCustomCommand(bitbakeDriver, command, 'Toaster')
```

Comment:
I wonder how this handles under kas. It likely breaks because the docker container gets deleted if the commmand exits. We could get away with just documenting it in this PR.

Reply:
BTW you could add some documentation in the README about toaster.

Reply:
Just added it about the commands. Do you want me to mention that it might not work in docker?

Reply:
Yes you can document that it's not compatible with docker. It would be very tricky to configure. I think the user can provide very specific docker container options to make it work but I don't know which ones. You could say:
"Note: Toaster requires additional packages and ports. It may not be compatible with all commandWrapper settings. Read the Toaster manual to configure them to have all the required dependencies."